docs/src/DatePicker.doc.js

```diff
@@ -0,0 +1,398 @@
+          'isRTL (Right-To-Left) provides support to specific locales by positioning textfield icon accordingly',
+      },
+      { name: 'label', type: 'string' },
+      {
+        name: 'localeData',
+        type: 'date-fns locale objects',
+        description: `DatePicker accepts imported locales from the open source date utility library date-fns. To use 'localeData', first, add the date-fns dependency in package.json. Second, use static or dynamic imports to get the locales from date-fns. Then, pass the locale to the localeData prop.`,
+        href: 'localeData',
+      },
+      {
+        name: 'maxDate',
+        type: 'Date',
+        description: 'End date for which range selection is enabled',
+      },
+      {
+        name: 'minDate',
+        type: 'Date',
+        description: 'Start date for which range selection is enabled',
+      },
+      {
+        name: 'onChange',
+        required: true,
+        type: '({event: SyntheticInputEvent<>, value: Date }) => void',
+      },
+      { name: 'placeholder', type: 'string' },
+      {
+        name: 'selectsEnd',
+        type: 'boolean',
+        description:
+          'Defines a date field input to select the end date in a date range. Required for date range picker.',
+        href: 'rangePicker',
+      },
+      {
+        name: 'selectsStart',
+        type: 'boolean',
+        description:
+          'Defines a date field input to select the start date in a date range. Required for date range picker.',
+        href: 'rangePicker',
+      },
+      {
+        name: 'startDate',
+        type: 'Date',
+        description:
+          'Start date for which selection is enabled. Required for date range picker.',
+        href: 'rangePicker',
+      },
+      { name: 'value', type: 'Date', description: 'Selected date value' },
+    ]}
+  />
+);
+
+card(
+  <Example
+    description="
+    Use this component to select date inputs.
+  "
+    name="Example: Basic date picker field"
+    defaultCode={`
+function DatePickerExample() {
+  const [date, setDate] = React.useState(new Date());
+  return (
+    <DatePicker
+      accessibilityLabelNextMonth='fakeaccessibilityLabelNextMonth'
+      accessibilityLabelPreviousMonth='fakeaccessibilityLabelPreviousMonth'
+      accessibilityLabelIcon='fakeAccessibilityLabelIcon'
+      id="example-basic"
+      onChange={({value}) => setDate(value)}
+      value={date}
+    />
+
+  )
+}
+`}
+  />
+);
+
+card(
+  <Example
+    id="rangePicker"
+    description="
+    You can use this component for a visual divider between two elements.
+  "
+    name="Example: Date range picker"
+    defaultCode={`
+function DatePickerRangeExample() {
+  const [startDate, setStartDate] = React.useState(undefined);
+  const [endDate, setEndDate] = React.useState(undefined);
+
+  const handleSetStartDate = ({ event, value }) => setStartDate(value);
+  const handleSetEndDate = ({ event, value }) => setEndDate(value);
+
+  return (
+    <Box>
+      <Box
```

Comment:
@AlbertCarreras Can we use `<Row gap={1} justifyContext="between" />`

Reply:
fixed